src/test/java/myApp/service/ClientServiceTest.java

```diff
@@ -0,0 +1,191 @@
+package myApp.service;
+
+import myApp.entity.Client;
+import myApp.entity.Order;
+import myApp.repository.ClientRepository;
+import myApp.repository.OrderRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.*;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+class ClientServiceTest {
+
+    @Mock
+    private ClientRepository clientRepository;
+
+    @Mock
+    private OrderRepository orderRepository;
+
+    @InjectMocks
+    private ClientService clientService;
+
+    @Captor
+    private ArgumentCaptor<Client> clientCaptor;
+
+    @Captor
+    private ArgumentCaptor<Order> orderCaptor;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
```

Comment:
if you use such approach you have to add method with @AfterEach to close the resource 
or

simplify the way of mock creation and injection (use @ExtendWith(MockitoExtension.class) as a class annotation and remove setUp method)